tasks/agent.py

```diff
@@ -19,6 +19,22 @@
 #constants
 BIN_PATH = os.path.join(".", "bin", "agent")
 AGENT_TAG = "datadog/agent:master"
+DEFAULT_BUILD_TAGS = [
```

Comment:
Why not just setting `exclude = ["docker", "kubelet", "kubeapiserver"] if invoke.platform.WINDOWS else ["kubeapiserver"]` abd keep the ALL_TAGS ?

Reply:
Because the DCA will probably introduce several other build tags, might as well do it right now.
Using `ALL_TAGS` for the infra agent was implying all other binaries were subsets of the agent (dsd, puppy), but that's not true anymore.